src/ontology/edits/oeo-physical.omn

```diff
@@ -3013,86 +2960,82 @@ Source: https://wiki.openmod-initiative.org/wiki/Transformer"@en,
 
     Annotations: 
         rdfs:label "transport demand"
     
     SubClassOf: 
-        OEO_00000127
+        <http://openenergy-platform.org/ontology/oeo/OEO_00000127>
     
     
-Class: OEO_00000425
+Class: <http://openenergy-platform.org/ontology/oeo/OEO_00000425>
 
     Annotations: 
         <http://purl.obolibrary.org/obo/IAO_0000115> "A turbine is an energy converting device that converts energy from a moving fluid flow into rotational energy."@en,
         rdfs:label "turbine"
     
     SubClassOf: 
-        OEO_00000011
+        <http://openenergy-platform.org/ontology/oeo/OEO_00000011>
     
     
-Class: OEO_00000429
+Class: <http://openenergy-platform.org/ontology/oeo/OEO_00000429>
 
     Annotations: 
         <http://purl.obolibrary.org/obo/IAO_0000115> "Underground hydrogen storage is the practice of hydrogen storage in underground caverns,salt domes and depleted oil/gas fields."@en,
         <http://purl.obolibrary.org/obo/IAO_0000119> "https://en.wikipedia.org/w/index.php?title=Underground_hydrogen_storage&oldid=895350780",
         rdfs:label "underground hydrogen storage"
     
     SubClassOf: 
-        OEO_00000061
+        <http://openenergy-platform.org/ontology/oeo/OEO_00000061>
     
     
-Class: OEO_00000430
+Class: <http://openenergy-platform.org/ontology/oeo/OEO_00000430>
 
     Annotations: 
         rdfs:label "undirected edge"
     
     SubClassOf: 
-        OEO_00000255
+        <http://openenergy-platform.org/ontology/oeo/OEO_00000255>
     
     
-Class: OEO_00000433
+Class: <http://openenergy-platform.org/ontology/oeo/OEO_00000433>
 
     Annotations: 
         rdfs:label "v2 grid"
     
     SubClassOf: 
-        OEO_00000421
+        <http://openenergy-platform.org/ontology/oeo/OEO_00000421>
     
     
-Class: OEO_00000437
+Class: <http://openenergy-platform.org/ontology/oeo/OEO_00000437>
 
     Annotations: 
-        <http://purl.obolibrary.org/obo/IAO_0000115> "Volatile organic compounds (VOCs) are portions of matter consisting of organic chemicals that have a high vapor pressure at ordinary room temperature. Their high vapor pressure results from a low boiling point, which causes large numbers of molecules to evaporate or sublimate from the liquid or solid form of the compound and enter the surrounding air, a trait known as volatility.
-
-[...]
-
-VOCs are numerous, varied, and ubiquitous. They include both human-made and naturally occurring chemical compounds.
-
-[...]
-
-Some VOCs are dangerous to human health or cause harm to the environment."@en,
-        <http://purl.obolibrary.org/obo/IAO_0000115> "https://en.wikipedia.org/w/index.php?title=Volatile_organic_compound&oldid=880677383",
+        <http://purl.obolibrary.org/obo/IAO_0000115> "A volatile organic compound (VOC) is a portion of matter consisting of organic compounds that is capable of producing photochemical oxidants by reactions with nitrogen oxides in the presence of sunlight. Hence it can at as an air pollutant."@en,
```

Comment:
```suggestion
        <http://purl.obolibrary.org/obo/IAO_0000115> "A volatile organic compound (VOC) is a portion of matter consisting of organic compounds that is capable of producing photochemical oxidants by reactions with nitrogen oxides in the presence of sunlight. Hence it can act as an air pollutant."@en,
```